src/cmd_handlers.c

```diff
@@ -1011,6 +1016,56 @@ goto_cmd(const cmd_info_t *cmd_info)
 	return 0;
 }
 
+/* Opens external editor to edit specified register contents. */
+static int
+regedit_cmd(const cmd_info_t *cmd_info)
+{
+	const int regname = cmd_info->argc > 0 ? tolower(cmd_info->argv[0][0]) : DEFAULT_REG_NAME;
+	if(regname == BLACKHOLE_REG_NAME)
+	{
+		ui_sb_err("Cannot modify blackhole register.");
+		return CMDS_ERR_CUSTOM;
+	}
+
+	reg_t *reg = regs_find(regname);
+	if(reg == NULL)
+	{
+		ui_sb_err("Register with given name does not exist.");
+		return CMDS_ERR_CUSTOM;
+	}
+	char tmp_fname[PATH_MAX + 1];
+	generate_tmp_file_name("vifm.regedit", tmp_fname, sizeof(tmp_fname));
+
+	mode_t saved_umask = umask(~0600);
+	const int write_result = write_file_of_lines(tmp_fname, reg->files, reg->nfiles);
+	if(write_result != 0)
+	{
+		ui_sb_err("Couldn't write register content into external file.");
+		return CMDS_ERR_CUSTOM;
+	}
+	const int edit_result = vim_view_file(tmp_fname, 1, 1, /*allow_forking=*/1);
+	if(edit_result != 0)
+	{
+		ui_sb_err("Register content edition went unsuccessful.");
+		return CMDS_ERR_CUSTOM;
+	}
+	(void)umask(saved_umask);
+
+	int read_lines = 0;
+	strlist_t *edited_content = read_file_of_lines(tmp_fname, &read_lines);
```

Comment:
The type should be `char **`.  You can run `./configure` with `--enable-developer` or `--enable-werror` to treat warnings as errors, otherwise they are easy to miss.  This however might break the build if your toolchain warns about code that's already in the `master`.

Reply:
> This however might break the build if your toolchain warns about code that's already in the `master`.

And it does.

Reply:
Well, this time it didn't cause any issues with `-werror`.